eip/EIPS/eip-4337.md

```diff
@@ -624,6 +624,219 @@ Returns an array of the entryPoint addresses supported by the client. The first
 }
 ```
 
+#### * eth_chainId
+
+Returns EIP155 Chain ID.
+
+```json=
+# Request
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "method": "eth_chainId",
+  "params": []
+}
+
+# Response
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": "0x1"
+}
+```
+
+### debug Namespace
+
```

Comment:
Maybe a few words here about why bundlers need to implement the debug namespace (required by the tests suite) and that these APIs shouldn't be available to users in production?

Reply:
Ok, I thought it was implied by the name itself, but I can succinctly describe it.